### PR TITLE
e2e tests header feature

### LINKED ISSUE #87: GCcollab - Header Feature Testing
Tests for subcomponents: 
1. Home Button
2. Network Button
3. Groups Button 
4. Jobs Button 
5. Support Button
6. Search Feature 
7. Messages Feature 
8. Notifications Feature 
9. Profile Feature 
10. Language Feature

### PR DESCRIPTION
**Pull request by AdiMakkar**
closes #87

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ const routes: Routes = [
     path: CoreRoutes.Home,
     title: translations.titles.home,
     loadChildren: () => import('./features/home/home.module').then(m => m.HomeModule),
-    canActivate: [InterceptorGuard],
+    //canActivate: [InterceptorGuard],
```
Comment (doug0102) [inline src/app/app-routing.module.ts:36]: I think you forgot to uncomment this.

**Review by doug0102 — DISMISSED** [review]
Looks good on the Angular side except we need the InterceptorGuard on the home route.

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (AdiMakkar) [reply src/app/app-routing.module.ts:36]: yes my apologies, ill uncomment it.

**Review by doug0102 — APPROVED** [review]

File: haibun-e2e-tests/package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      "test-webkit": "HAIBUN_SPLIT_SHARED=a-browser=webkit haibun-cli local",
+      "test-local": "HAIBUN_SPLIT_SHARED='a-browser=chromium,webkit,firefox,chromium.iPhone 6,chromium.iPad (gen 7) landscape,chromium.iPad Mini,chromium.Pixel 5,chromium.iPhone X landscape,chromium.Kindle Fire HDX,chromium.Moto G4 landscape' haibun-cli local",
+      "test-envc": "HAIBUN_ENVC=what=what1,what=what2,also=also1,also=also2 haibun-cli envc",
+      "test-envc-10": "HAIBUN_LOOPS=2 HAIBUN_MEMBERS=5 HAIBUN_ENVC=what=what1,what=what2,also=also1,also=also2 haibun-cli envc"
+    },
+    "keywords": [],
+    "author": "vid",
+    "license": "ISC",
+    "dependencies": {
+      "@haibun/cli": "^1.25.0",
```
Comment (vid) [inline haibun-e2e-tests/package.json:26]: these should be updated to the version with tab recognition (1.28.12)

**Review by vid — COMMENTED** [review]
looks good but needs a library upgrade

**Review by doug0102 — APPROVED** [review]

**Pull request merged**